packages/unchained-client/src/index.ts

```diff
@@ -4,5 +4,7 @@ export * from './evm'
 export * as ws from './websocket'
 
 export * as bitcoin from './bitcoin'
+export * as dogecoin from './dogecoin'
+// export * as ethereum from './ethereum'
```

Comment:
```suggestion
export * as ethereum from './ethereum'
```

Reply:
I feel like this came through a merge/rebase. will take a look